.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+     {
+        "name": "Remote",
+        "type": "go",
+        "request": "launch",
+        "mode": "remote",
+        "remotePath": "${workspaceRoot}",
+        "port": 2345,
+        "host": "127.0.0.1",
+        "program": "${workspaceRoot}",
+        "env": {}
+      },
+    ]
+}
```

Comment:
This file is your personal file, we may not need to commit it.

If you really want to commit it:

Since it's JSON format, the // comment should not be added, otherwise, GitHub will report red error. And missing new line.